plugins/catalog-import/api-report.md

```diff
@@ -146,6 +147,11 @@ export function defaultGenerateStepper(
   defaults: StepperProvider,
 ): StepperProvider;
 
+// Warning: (ae-missing-release-tag) "DefaultImportComponentPage" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const DefaultImportComponentPage: () => JSX.Element;
```

Comment:
it's unfortunate to use the word "component" here since it's just one of several possible kinds

Reply:
Agree, I will rename it. Even thought everything else is still referring to components. Like "Register an existing component" or "Start tracking your component in …" 😆